python/tvm/tir/op.py

```diff
@@ -271,11 +270,11 @@ def call_llvm_intrin(dtype, name, *args, span=None):
     else:
         llvm_id = name
     if llvm_id == 0:
-        warnings.warn(f"Unknown llvm intrinsic function {name}, falling back to 0")
+        raise ValueError(f"Unknown llvm intrinsic function {name}")
```

Comment:
nit: I think it would be good to add a test for this change

Reply:
Added a test case.